.ci/ci-maven-repair.sh

```diff
@@ -4,11 +4,6 @@
 set -e
 export M2_HOME=/usr/local/maven
 
-VERSION=`xmlstarlet sel -t -v '//_:project/_:properties/_:revision' src/pom.xml`
-sed -i -e 's/\${revision}/'$VERSION'/' src/repairnator-core/pom.xml
-sed -i -e 's/\${revision}/'$VERSION'/' src/pom.xml
-sed -i -e 's/\${revision}/'$VERSION'/' "${TEST_PATH}"/pom.xml
-
 NPEFIX_VERSION=`xmlstarlet sel -t -v '//_:dependency[_:artifactId="npefix"]/_:version' src/maven-repair/pom.xml`
```

Comment:
How to handle this if we don't xmlstarlet?

Reply:
That's what the plugin I introduced does (flatten-maven-plugin). See [maven docs](https://maven.apache.org/maven-ci-friendly.html#install-deploy).

If you also take a look at [Jenkins logs](https://ci.eclipse.org/repairnator/job/repairnator-ci/job/PR-1156/2/execution/node/72/log/?consoleFull), you can see that it replaces `${revision}` by the value defined in the homonymous property. It wouldn't be able to install (at least in both my PCs and Ubuntu VM it didn't work, @cokefh wasn't able to reproduce) without this, or running the sed commands.

Reply:
I'm just pointing that there is still a call to `xmlstarlet` in line 12

Reply:
Oh sorry, I got confused with the other email I got. Did you delete any comment?

But yes, I forgot there was still that call to `xmlstarlet`. I'll change that line. 
Also, I don't really understand why we need to pass `NPEFix`s version through the cli, so maybe that could be removed as well.

Reply:
I don't know either, I'm not really familiar with that module, maybe @cokefh can tell us?